Cargo.toml

```diff
@@ -5,7 +5,7 @@ keywords = ["geospatial", "geodesy", "cartography", "geography"]
 categories = ["science"]
 license = "MIT OR Apache-2.0"
 version = "0.10.0"
-rust-version = "1.65"  # Uses the let-else construct introduce in 1.65
+rust-version = "1.65"                                                                  # Uses the let-else construct introduce in 1.65
```

Comment:
When touching up this line, you might as well correct my speelling missteak: "introduce" should be "introduced"